main/src/ca/uwaterloo/flix/language/dbg/DocAst.scala

```diff
@@ -204,6 +204,15 @@ object DocAst {
     def ArrayStore(d1: Expr, index: Expr, d2: Expr): Expr =
       Assign(SquareApp(d1, List(index)), d2)
 
+    def StructNew(sym: Symbol.StructSym, exps: List[(Name.Label, Expr)], d2: Expr) =
+      DoubleKeyword("new " + sym.toString, exps.foldRight(RecordEmpty: Expr){ case (cur, acc) => RecordExtend(cur._1, cur._2, acc)}, "@", Left(d2))
```

Comment:
Maybe one or more local variables would be reasonable here.

Also the return type of the function is missing.